letsencrypt/data/ir_cron.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data noupdate="1">
+        <record id="cronjob" model="ir.cron">
+            <field name="name">Update letsencrypt certificates</field>
+            <field name="interval_type">weeks</field>
+            <field name="interval_number">11</field>
+            <field name="numbercall">-1</field>
+            <field name="model">letsencrypt</field>
+            <field name="function">cron</field>
+            <field name="nextcall">2016-01-01</field>
```

Comment:
I don't remember...
I just saw the following example [odoo/mail]( https://github.com/odoo/odoo/blob/96e085355d2e29d81ace06e08dc76ea5668846ef/addons/mail/data/mail_data.xml#L23)
If you consider what don't affect then is good for me too